src/vm/compile.cpp

```diff
@@ -1355,30 +1355,6 @@ BOOL CanDeduplicateCode(CORINFO_METHOD_HANDLE method, CORINFO_METHOD_HANDLE dupl
     DynamicMethodDesc * pMethod = GetMethod(method)->AsDynamicMethodDesc();
     DynamicMethodDesc * pDuplicateMethod = GetMethod(duplicateMethod)->AsDynamicMethodDesc();
 
-    //
```

Comment:
Why is this code removed?

Reply:
This code was needed before the return type was captured in GCInfo. Now that it is captured in GCInfo, this is not needed anymore. (Also, this method is only for fragile NGen that is not used in .NET Core anymore.)